.github/workflows/ci.yml

```diff
@@ -34,14 +34,14 @@ jobs:
       - name: Build Framework
         run: swift package generate-xcodeproj --output generated/ && carthage build --project-directory generated/ --verbose --no-skip-current
   spm-11:
-    name: SPM Build Xcode 11
-    runs-on: macOS-10.15
+    name: Build Xcode 11
+    runs-on: macOS-11
     strategy:
       matrix:
         platforms: [
           'iOS_12,tvOS_12',
           'iOS_13,tvOS_13,watchOS_6',
-          'macOS_10_15'
+          'macOS_10_15',
```

Comment:
Fun fact, we can still run tests against the macOS 10.15 SDK even while running on macOS 11. We can only do this while running on Xcode 11.